kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/importer/DataRecordMerger.java

```diff
@@ -119,21 +119,22 @@ private void mergeUpdate(final DataRecord dataRecord, final Map<DataRecord.Key,
     }
     
     private void mergeDelete(final DataRecord dataRecord, final Map<DataRecord.Key, DataRecord> dataRecords) {
-        DataRecord beforeDataRecord = dataRecords.get(dataRecord.getKey());
```

Comment:
Could we remove `mergeDelete` and `merge` methods etc

Reply:
I'll think about it later